src/main/cpp/v8/JNIV8Object.cpp

```diff
@@ -355,7 +355,7 @@ jobject JNIV8Object::jniCallV8MethodWithReturnType(JNIEnv *env, jobject obj, jst
                 break;
             case JNIV8MarshallingError::kOutOfRange:
                 ThrowJNICastError("return value '"+
-                                  JNIV8Marshalling::v8string2string(resultRef->ToString())+"' is out of range for method '" + strMethodName + "'");
+                                  JNIV8Marshalling::v8string2string(resultRef->ToString(context).ToLocalChecked())+"' is out of range for method '" + strMethodName + "'");
```

Comment:
Maybe we could have a "v8value2string" that could accept also a MaybeLocal as a possible overload; this could then do this conversion, check if the returned maybelocal<string> is not empty, return "" otherwise, etc. Would even safe some code and be "safer".